ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/proxy/https/cucushift-installer-rehearse-vsphere-upi-proxy-https-workflow.yaml

```diff
@@ -11,6 +11,7 @@ workflow:
     - ref: upi-conf-vsphere-ova
     - ref: upi-install-vsphere
     - ref: ipi-install-times-collection
+    - ref: enable-qe-catalogsource
```

Comment:
I think we can replace all the pre steps with `cucushift-installer-rehearse-vsphere-upi-proxy-https-provision` chain, and all the post steps with `cucushift-installer-rehearse-vsphere-upi-proxy-https-deprovision` chain.